examples/chip-tool/commands/pairing/OpenCommissioningWindowCommand.h

```diff
@@ -40,6 +40,7 @@ class OpenCommissioningWindowCommand : public CHIPCommand
                     &mIteration, "Number of PBKDF iterations to use to derive the verifier.  Ignored if 'option' is 0.");
         AddArgument("discriminator", 0, 4095, &mDiscriminator, "Discriminator to use for advertising.  Ignored if 'option' is 0.");
         AddArgument("timeout", 0, UINT16_MAX, &mTimeout, "Time, in seconds, before this command is considered to have timed out.");
+        AddArgument("joint-fabric", 0, 1, &mJointFabric, "Open Joint Fabric Commissioning Window.");
```

Comment:
For consistency lets please use `--jcm` as the pairing flag.

Reply:
The change has been reverted.